python-flask-example-heroku/config.py

```diff
@@ -0,0 +1,21 @@
+import os
+
+
+class Config(object):
```

Comment:
Use class declaration syntax form Python 3:

```suggestion
class Config:
```